.github/workflows/test.yml

```diff
@@ -15,7 +15,7 @@ on:
       - docs/**
 
 jobs:
-  test:
+  test-provider:
```

Comment:
this is confusing, the tests are of type integration, can we figure out a better name?

Reply:
maybe just `integration` ? My point is that we don't need a prefix, since in any PR status, GitHub prepends the Name of the workflow (currently `Test` anyway)

Reply:
Changed to `test-integration`.